internal/api/grpc/action/v2beta/query.go

```diff
@@ -174,7 +174,7 @@ func targetFieldNameToSortingColumn(field *action.TargetFieldName) query.Column
 	}
 	switch *field {
 	case action.TargetFieldName_TARGET_FIELD_NAME_UNSPECIFIED:
-		return query.TargetColumnID
+		return query.TargetColumnCreationDate
```

Comment:
I think this fixes a bug

Reply:
as this is totally unrelated to the PR topic and probably needs to be backported, please create a separate PR

Reply:
moved to https://github.com/zitadel/zitadel/pull/9795